src/web/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "eslint-plugin-react": "^7.37.1",
+    "normalize.css": "^8.0.1",
```

Comment:
Not sure if this is necessary. Cold we just add this to the main css file?
```
* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}
```

Reply:
Sure we can do that . I removed the package and added the reset rule in the start of index.css
My idea was to use the normalize.css that seems to add a variaty of reset rules for different browsers. But it adds to complexity and probably we dont need it atm:

https://github.com/necolas/normalize.css/blob/master/normalize.css

Reply:
Ivan and I discussed this. We agreed to revert back to normalize.css out of the box.

@akiskips You can resolve this comment when you've done that. Thanks!

Reply:
Reverted to the normalize.css package.